bench/db_bench.cc

```diff
@@ -895,7 +1045,24 @@ int main(int argc, char **argv) {
 
     // Run benchmark against default environment
     g_env = leveldb::Env::Default();
-    Benchmark benchmark;
+
+    BenchmarkLogger *logger;
+    if(FLAGS_logger != CSV)
```

Comment:
This is like double negation ;d Why not `if(FLAGS_logger) logger = new CsvLogger()`?